### PR TITLE
Add March meetup files

### PR DESCRIPTION
**Pull request by katzien**
- added the event file
- added James's file
- added James's talk

### CODE REVIEW

File: data/talks/2018/03/crafting-quality-php-applications-an-overview.json
Comment (katzien) [inline data/talks/2018/03/crafting-quality-php-applications-an-overview.json]: @DaveLiddament @opdavies Question :) Given potential event name/slug clashes (e.g. "Lightning talks"), should we go with the implicit assumption that we'll look in a matching dir for the event file? Ie. if the talk file is in 2018/03, we look for a matching event under the events/2018/03 dir?

Otherwise we'll need to somehow specify a full path to the event file, but I don't think that's necessary? I think assuming a matching <year>/<month> dir for talks and events should be fine? Just wondering if it will work for you implementation-wise, or would you prefer a different structure in the json files?

**Comment by katzien** [comment]
@DaveLiddament CI / run.sh fails if I have those fields in the talk json file:
```
"joindinUrl": "96cee",
"slidesUrl": "",
"videoUrl": ""
```

If I remove them run.sh returns no errors. With them, I get this error:

`Error importing [crafting-quality-php-applicati] of type [Phpsw\Website\Entity\Talk]. Field []: This form should not contain extra fields.`

What do I need to do to make the validator accept those fields? :)

File: data/talks/2018/03/crafting-quality-php-applicati.json
Comment (DaveLiddament) [inline data/talks/2018/03/crafting-quality-php-applicati.json]: Convention we've used elsewhere is a dash to separate words rather than camel case:

joindin-url

**Review by DaveLiddament — DISMISSED** [review]

File: data/talks/2018/03/crafting-quality-php-applicati.json
Comment (katzien) [reply data/talks/2018/03/crafting-quality-php-applicati.json]: ah yeah cool with me, I just took this from the Talks class as it was :)

**Comment by katzien** [comment]
Well I can't merge this in myself so if you're happy with this Dave then merge it in. Thanks for doing the fixes!

**Review by DaveLiddament — APPROVED** [review]

**Pull request merged**
